modules/ROOT/pages/6.4.1-release-notes.adoc

```diff
@@ -0,0 +1,1334 @@
+
+To fix this issue, a conditional filter was added which identifies, if an element is either a `contenteditable="true"` element or a `contenteditable="false"` element to its decendants.  If the element matches as a `contenteditable="false"` element to itself or decendants, the `visual character` plugin will remove the highlighted selecter from within the `dom` viewport.
+
+As a result, in {productname} 6.4.1, visual characters no longer render inside `contenteditable="false"` elements.
+
+=== Lists in a noneditable root were incorrectly editable using list API commands, toolbar buttons and menu items
+//#TINY-9458
+
+In previous versions of {productname}, a bug was found that affected the *lists*  and *Advanced List* plugins when the {productname} editor body was set to `contenteditable = "false"`. The consequence meant that the lists, that were even set as `contenteditable="false"`,  elements could be modified by using list API commands, toolbar buttons and menu items.
+
+The fix involved implementing a new process to  check the element conditions.  The check now looks to identify if the *list* or *Advanced List* is a `contenteditable = "false"` element.
+
+As a result, in {productname} 6.4.1, *list* or *Advanced List*  plugins that are set as `contenteditable = "false"` elements can no longer be modified by using list API commands, toolbar buttons and menu items.
+
+=== Color picker dialog would not update the preview color if the hex input value was prefixed with # symbol
+//#TINY-9457
+
+In previous versions of {productname}, the preview sample box within the color picker dialog would not render. This issue could be replicated when a hex `value` was prefixed with a `#` symbol.
+
+Example of a `incorrect` vs `correct` value input:
+
+* `#BDE123`: Incorrect hex value input.
+* `BDE123`: Correct hex value input.
+
+In order to fix this bug, {productname} 6.4.1, not detects if the user inserts a `#` symbol before the hex `value`, which removes the leading `#` prefix before rendering to the color picker component.
+
+In {productname} 6.4.1, the color picker works as expected when using hex `values` with the `#` prefix.
+
+=== Table cell selection was possible even if the element was in a noneditable root element
+//#TINY-9459
+
+Previous versions of {productname}, table cells set with the `contenteditable="false"` element properties could still be modified. This was because the `contenteditable="true"` element property settings were set to `false` in the root. This meant that any table UI actions, commands or table selections could still affect any tables that were within that specific `contenteditable="false"` element root.
+
+{productname} 6.4.1, introduced a blocker that is executed when tables are inside a `contenteditable="false"` element root.
+
+In {productname} 6.4.1, all table UI actions, commands and selections have been `disabled` for tables when either the table or the editor body have been set as `contenteditable="false"` elements.
```

Comment:
```suggestion
In previous versions of {productname}, table cells set within a `contenteditable="false"` root element could still be modified. This was due to configuration settings within the `contenteditable="true"` function as it was set to `false` by default in the root. This meant that any table UI actions, commands or selections could affect tables that were nested within the `contenteditable="false"` root element.
```

Reply:
@tiny-ben-tran - moved this suggestion here, as the previous suggestion was outdated.
Please review when possible.

Reply:
```suggestion
In previous versions of {productname}, table cells set within a `contenteditable="false"` root element could still be selected and modified. This meant that any table UI actions, commands or selections could affect tables that were nested within the `contenteditable="false"` root element.

In {productname} 6.4.1, table is no longer selectable or editable by table UI and commands, when either the table or the editor body have been set as `contenteditable="false"` elements.
```